src/Core/Geographic/Crs.js

```diff
@@ -2,38 +2,43 @@ import proj4 from 'proj4';
 
 proj4.defs('EPSG:4978', '+proj=geocent +datum=WGS84 +units=m +no_defs');
 
-const TMS = [
-    'WMTS:WGS84',
-    'WMTS:PM',
-];
+const root_tms = 'TMS';
+const root_epsg = 'EPSG';
+const nCrs = ['4326', '3857'];
 
-const EPSG = [
-    'EPSG:4326',
-    'EPSG:3857',
-];
+const TMS = nCrs.map(n => `${root_tms}:${n}`);
+const EPSG = nCrs.map(n => `${root_epsg}:${n}`);
+
+function isTms(crs) {
+    return crs.includes(root_tms);
+}
+
+function isEpsg(crs) {
+    return crs.includes('EPSG');
```

Comment:
See, this is not coherent at all, following your initial idea, you should use `root_epsg`.